### ISSUE TITLE
Work on frontend

### ISSUE THREAD

**Issue by 035WasTaken** [JavaRockMC/javarock-site#2]
Frontend needs to be worked on. I have already set it up for the most part (disregard CSS, just delete that) and will be working on the backend.
If you aren't sure if something will "mesh" properly with the backend, ask me and I'll take a look.

**Comment by 035WasTaken** [comment]
i get that putting "wont fix" as a label is funny, but seriously, I cannot work more on this until the frontend is worked on to the point of actual requests being made. you can model it however you'd like, but it would be nice to have this done sometime before christmas :)

**Comment by dudebehinddude** [comment]
waiting on #4
